pilot/pkg/networking/grpcgen/testdata/xds_bootstrap.json

```diff
@@ -2,10 +2,10 @@
   "xds_servers": [
     {
       "server_uri": "localhost:14057",
-      "channel_creds": [{"type": "insecure"}]
+      "channel_creds": [{"type": "insecure"}],
+      "server_features" : ["xds_v3"]
```

Comment:
This is needed because the new grpc-go expects server_features to be child of xds_servers https://github.com/grpc/grpc-go/pull/4087